content/boilerplates/notes/1.2.md

```diff
@@ -0,0 +1,75 @@
+
+## General
+
+- **Added** `traffic.sidecar.istio.io/includeInboundPorts` annotation to eliminate the need for service owner to declare `containerPort` in the deployment yaml file.  This will become the default in a future release.
+- **Added** IPv6 experimental support for Kubernetes clusters.
+
+## Traffic Management
+
+- **Improved** [locality based routing](/docs/ops/traffic-management/locality-load-balancing/) in multicluster environments.
+- **Improved** outbound traffic policy in [`ALLOW_ANY` mode](/docs/reference/config/installation-options/#global-options). Traffic for unknown HTTP/HTTPS hosts on an existing port will be [forwarded as is](/docs/tasks/traffic-management/egress/egress-control/#envoy-passthrough-to-external-services). Unknown traffic will be logged in Envoy access logs.
+- **Added** support for setting HTTP idle timeouts to upstream services.
+- **Improved** Sidecar support for [NONE mode](/docs/reference/config/networking/v1alpha3/sidecar/#CaptureMode) (without iptables) .
+- **Added** ability to configure the [DNS refresh rate](/docs/reference/config/installation-options/#global-options) for sidecar Envoys, to reduce the load on the DNS servers.
+- **Graduated** [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/) from Alpha to Alpha API and Beta runtime.
+
+## Security
+
+- **Improved** extend the default lifetime of self-signed Citadel root certificates to 10 years.
+- **Added** Kubernetes health check prober rewrite per deployment via `sidecar.istio.io/rewriteAppHTTPProbers: "true"` in the `PodSpec` [annotation](/docs/ops/setup/app-health-check/#use-annotations-on-pod).
+- **Added** support for configuring the secret paths via `global.sds.customTokenDirectory` for Istio mutual TLS certificates. Refer [here](https://github.com/istio/istio/issues/11984) for more details.
```

Comment:
```suggestion
- **Added** support for configuring the secret paths for Istio mutual TLS certificates. Refer [here](https://github.com/istio/istio/issues/11984) for more details.
```

Reply:
`global.sds.customTokenDirectory` is to configure the JWT token loading path. Not related to this feature.